tests/test/dashboard.test.js

```diff
@@ -1,12 +1,21 @@
 const url = require('../helpers/url').main
+const wait = require('../helpers/wait')
+
+/**
+ * Helper function to list locks on the dashboard
+ */
+const listLocksOnPage = async () =>
+  page.evaluate(() => {
+    const lockElements = document.getElementsByClassName('lock')
+    return Array.from(lockElements).map(el => el.getAttribute('data-address'))
+  })
 
 describe('The Unlock Dashboard', () => {
   beforeAll(async () => {
-    await page.goto(url('/dashboard'))
+    await page.goto(url('/dashboard'), { waitUntil: 'networkidle2' })
```

Comment:
`networkidle2` is important because it means not only do we "go" to the page, but we actually wait for all network calls to resolve.